spec/factories/prisons.rb

```diff
@@ -26,6 +26,10 @@
       '01154960%03d' % n
     end
 
+    adult_age do
+      10 + rand(9)
```

Comment:
The adult age can be any integer between 10 & 19? Can a 10 year old be classed as an adult?